pgbot/commands/admin/emsudo.py

```diff
@@ -1575,19 +1576,30 @@ async def cmd_emsudo_get(
                         1,
                     )
                 embed_dict = embed.to_dict()
-
+                pop_embed_dict = embed_utils.copy_embed_dict(
+                    embed_dict
+                )  # circumvents discord.py bug
+                corrected_embed_dict = None
+                corrected_pop_embed_dict = None
                 if embed_mask_dict:
                     if "fields" in embed_dict and "fields" in embed_mask_dict:
                         field_list = embed_dict["fields"]
                         embed_dict["fields"] = {
                             str(i): field_list[i] for i in range(len(field_list))
                         }
+                        field_list = pop_embed_dict["fields"]
+                        pop_embed_dict["fields"] = {
+                            str(i): field_list[i] for i in range(len(field_list))
+                        }
+
+                        print("pop_embed_dict (before filter):", pop_embed_dict)
```

Comment:
looks like a debug print, so I guess it needs to be removed

Reply:
D'oh!

Reply:
fixed